src/components/document/SubTabs/components/Text/Text.tsx

```diff
@@ -5,5 +5,5 @@ import { useStyles } from './Text.styles'
 export const Text: FC<{ content: string }> = ({ content }) => {
     const { classes } = useStyles()
 
-    return !content ? <i>No text</i> : <pre className={classes.preWrap}>{content.trim()}</pre>
+    return !content ? <i>No text</i> : <pre className={classes.preWrap} dangerouslySetInnerHTML={{ __html: content.trim() }} />
```

Comment:
there is already `react-highlight-words` component in the project, isn't it possible to use here?

Reply:
yes it would be possible, but since we already have to parse the string to get the results count and so on. We already have the highlights matched in the store. So i just have to insert them here, and this approach does not affect the other usages of Text.

If you really insist, I can change to use the npm package, but it would not change much in how the highlighting behaves